tuning/trackers/aimstack_tracker.py

```diff
@@ -20,6 +23,49 @@
 from .tracker import Tracker
 from tuning.config.tracker_configs import AimConfig
 
+AIM_HASH_EXPORT_FILENAME = "aim_run_hash"
+
+
+class CustomAimCallback(AimCallback):
+    """
+    Custom Aimstack callback is used to export run id from Aim
+    as soon as its created, which is on_init_end.
+    """
+
+    # path where we export run hash generated by Aim
+    # This is used to link back to the expriments from outside aimstack
+    hash_export_path = None
```

Comment:
Just thinking out loud, can we use the same variable name `aim_run_id_export_path` for ease of understanding?